submissions/Lu-sure/memory-pair-game/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+    padding: 0;
+    margin: 0;
+    box-sizing: content-box;
+    font-family: 'Langar', cursive;
+    font-size: 18px;
+}
+
+.navig {
+    background-color:  #53709b;
+    min-height: 70px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.new-game-button {
```

Comment:
Add `cursor: pointer;` for the `hover` state here as well

Reply:
Done.